CHANGELOG.md

```diff
@@ -6,6 +6,14 @@ Find changes for the upcoming release in the project's [changelog.d directory](h
 
 <!-- scriv-insert-here -->
 
+## 6.1.0 (2024-06-05)
+
+### Other changes
+
+- Jupyterhub 5.0.0 adopted as underlying Hub implementation.
+
+- Move from `pip` to `uv` in `nox` and CI.
+
```

Comment:
So, the idea behind how scriv is supposed to work is that we leave the fragments as fragments on all the feature branches. Then, when we're ready to make a release, we make a *new* branch, run `scriv collect` there (and any other release tasks like `make update` / `nox -s update-deps`), and then merge that right before the release. That release prep branch doesn't require review.

No need to move these back into fragments to redo this, just a note for next time, but note that there's now another change going into this release so now `scriv collect` won't work in that other branch, which is why `scriv collect` should be the last thing before a release on its own branch with only release stuff.

Reply:
Easy enough to revert, though.